### PR TITLE
Reveal new sample db tables by default

### PR DESCRIPTION
**Pull request by ranquild**
Epic https://github.com/metabase/metabase/issues/24807

How to test:
- Go to Admin -> Databases -> Sample and trigger manual sync. New sample tables should become visible.
- Re-install Metabase and complete setup. New sample table should be visible immediately. 
- Run a cypress test. New tables should still be hidden.

Screenshot 2022-11-15 at 15 56 50

### CODE REVIEW

**Review by cdeweyx — APPROVED** [review]

File: frontend/test/snapshot-creators/default.cy.snap.js [hunk 1]
```diff
@@ -227,6 +228,17 @@ describe("snapshots", () => {
...
+    ACCOUNTS_ID,
+    ANALYTIC_EVENTS_ID,
+    FEEDBACK_ID,
+    INVOICES_ID,
+  }) {
+    [ACCOUNTS_ID, ANALYTIC_EVENTS_ID, FEEDBACK_ID, INVOICES_ID].forEach(id => {
+      cy.request("PUT", `/api/table/${id}`, { visibility_type: "hidden" });
+    });
+  }
+
```
Comment (nemanjaglumac) [inline frontend/test/snapshot-creators/default.cy.snap.js:241]: 👌

**Review by gusaiani — COMMENTED** [review]
@ranquild, trying the branch locally.

A fresh install works all right.

Doing it on an existing install isn't working for me.

My steps:
Go to http://localhost:3000/admin/databases
Click on the Sample Database
Click on "Sync database schema now"
Wait a bit.

Also clicked on Rescan field values now.
Clicked on Save Changes.
Restarted the server after switching to this branch.

What am I missing?

**Comment by nemanjaglumac** [comment]
I can confirm the same behavior that @gusaiani experienced.

**Comment by ranquild** [comment]
I'll take a look

**Comment by ranquild** [comment]
@nemanjaglumac @gusaiani it works if you update from 0.44

**Comment by gusaiani** [comment]
@ranquild Thanks.

I tried setting up Sample Data from branch `release-x.44.x` and then going to this branch and following the steps above.

Still no dice.

Should I have done it another way?

**Comment by ranquild** [comment]
@gusaiani you might need to wait for like 1 minute after trigging the sync. And you'd also need to refresh the app to see new tables.

**Comment by ranquild** [comment]
Please let me know if the issue still happens and I'll fix it in the next PR

**Comment by gusaiani** [comment]
> Please let me know if the issue still happens and I'll fix it in the next PR

@ranquild Actually it doesn't happen, you're right, it's a matter of waiting enough.

I suppose the issue is different, that the button label `Sync triggered!` is removed a lot sooner than the sync ends.

**Pull request merged**